libs/template/spec/render-strategies/render-strategies/native.strategy.spec.ts

```diff
@@ -0,0 +1,82 @@
+// tslint:disable-next-line:nx-enforce-module-boundaries
+import { mockConsole } from '@test-helpers';
+import { getStrategies } from '@rx-angular/template';
+import { getMockNativeStrategyConfig } from '../../fixtures';
+import { from, of } from 'rxjs';
+import createSpy = jasmine.createSpy;
+
+const nativeStrategyName = 'native';
```

Comment:
If we go with a generic approach, maybe just `strategyName` / `testedStrategyName` ?

Reply:
Will change it!

Reply:
Done!